src/components/material/MaterialView.tsx

```diff
@@ -118,9 +119,8 @@ export class MaterialView extends Component<MaterialViewProps, MaterialViewState
                         components={{
                             IndicatorSeparator: null,
                             Option: (props) => {
+                                props.selectProps.styles.option = getCustomOptionSelectionStyle({ direction: 'none' })
```

Comment:
use the Option from ReactSelect.utils instead of doing this here.

Reply:
custom Option function is already be created in this file. Can we pass custom function inside Option component of react.util ?

Reply:
Okay. As discussed, let it be as is.